kernel/memory/src/paging/mapper.rs

```diff
@@ -1035,15 +1064,20 @@ impl<T: FromBytes> BorrowedMappedPages<T, Mutable> {
     }
 }
 
-impl<T: FromBytes, M: Mutability> BorrowedMappedPages<T, M> {
+impl<T: FromBytes, M: Mutability, B: Borrow<MappedPages>> BorrowedMappedPages<T, M, B> {
     /// Consumes this object and returns the inner `MappedPages`.
-    pub fn into_inner(self) -> MappedPages {
+    pub fn into_inner(self) -> B {
         self.mp
     }
+
+    /// Returns a reference to the inner borrowed `MappedPages`.
```

Comment:
It doesn't return a reference to the inner borrowed `MappedPages`, but rather to the container holding the `MappedPages`.

Reply:
true, thanks.